packages/core/webpack/dev-client.js

```diff
@@ -11,15 +11,16 @@ const appPath = path.join(appDir, "app");
 const loaderPath = require.resolve("./config/loaders");
 delete require.cache[loaderPath];
 const commonLoaders = require(loaderPath);
+const context = path.join(__dirname, "../src");
 
 process.traceDeprecation = true;
 
 /** @type {import("webpack").Configuration} */
 module.exports = {
-  devtool: "eval",
+  devtool: "source-map",
```

Comment:
ooooo what's this do? jeje

Reply:
Oh! it was a test. 
This helps us to build the blueprint components correctly with their proper `.map` file. I was tired of all of those annoying  warning messages. -just in dev mode btw-. It is not webworker related. I'll double check if it worth it.